catalog/tests/dags/database/staging_database_restore/test_staging_database_restore.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime, timedelta
+from unittest import mock
+
+import pytest
+from airflow import DAG
+from airflow.providers.amazon.aws.hooks.rds import RdsHook
+from airflow.utils.trigger_rule import TriggerRule
+
+from catalog.tests.dags.database.staging_database_restore.data import (
+    DESCRIBE_DB_INSTANCES_RESPONSE,
+)
+from database.staging_database_restore import staging_database_restore
+
+
+REFERENCE_DATE = datetime(2021, 1, 1, 0, 0, 0)
+
+
+@pytest.fixture
+def mock_rds_hook() -> RdsHook:
+    return mock.MagicMock(spec=RdsHook)
+
+
+@pytest.mark.parametrize("should_skip", [True, False])
+def test_skip_restore(should_skip):
+    with mock.patch.object(
+        staging_database_restore.slack, "send_message"
+    ) as mock_send_message:
+        actual = staging_database_restore.skip_restore.function(should_skip)
+        assert actual == (not should_skip)
```

Comment:
Should we add tests that mock the `SKIP_STAGING_DATABASE_RESTORE` variable?

Reply:
I think since that's a `Variable.get` with a simple `or` comparison between that and `should_skip`, that testing the conditions there is sufficient. This seemed easier than mocking the `Variable.get` step, but I can change it to that kind of test and remove the `should_skip` param if you think that's best!